### PR TITLE
Issue 377 hidden user

### PR DESCRIPTION
**Pull request by wtm99**
Issue #377

### CODE REVIEW

File: BLUESPAWN-win-client/src/hunt/hunts/HuntT1136.cpp [hunk 1]
```diff
@@ -41,6 +42,19 @@ namespace Hunts {
         }
         SUBSECTION_END();
 
+        SUBSECTION_INIT(HIDDEN_USER, Normal);
```
Comment (ION28) [inline BLUESPAWN-win-client/src/hunt/hunts/HuntT1136.cpp:45]: Propose moving this to Cursory due to speed of check and high likelihood of maliciousness

**Review by ION28 — CHANGES_REQUESTED** [review]
See comment. Rest of the pr looks good to me

File: BLUESPAWN-win-client/src/hunt/hunts/HuntT1136.cpp (hunk 1 above)
Comment (wtm99) [reply BLUESPAWN-win-client/src/hunt/hunts/HuntT1136.cpp:45]: Ok

**Comment by ION28** [comment]
Tested 
image

**Review by ION28 — APPROVED** [review]

**Pull request merged**